scripts/build-binary.sh

```diff
@@ -6,7 +6,13 @@ source ./scripts/test_lib.sh
 
 VER=$1
 REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd.git}"
-
+# When ${CURRENT_BRANCH} is 0, the scripts doesn't clone the remote
```

Comment:
it should be `1`? if the `CURRENT_BRANCH` has been set, should we log the `VER` will be ignored?

Reply:
Good catch. updated.